resources/views/components/navbar.blade.php

```diff
@@ -15,17 +15,13 @@
                     @endauth
                 </li>
                 <li>
-                    <a href="{{ route('home') }}"
+                    <a href="{{ route('home') }}" id="scrollToHomeBtn"
```

Comment:
Pour rester coherent avec les classes, j'utiliserai kebab-case pour les id aussi `scroll-to-home-btn`

Reply:
I forgot to delete completely this id. We do not need it because there is a laravel route that takes the lead and charges the page.